cmd/argocd-server/commands/root.go

```diff
@@ -97,6 +101,8 @@ func NewCommand() *cobra.Command {
 	command.Flags().StringVar(&dexServerAddress, "dex-server", common.DefaultDexServerAddr, "Dex server address")
 	command.Flags().BoolVar(&disableAuth, "disable-auth", false, "Disable client authentication")
 	command.AddCommand(cli.NewVersionCmd(cliName))
+	command.Flags().IntVar(&listenPort, "listen-port", common.DefaultPortAPIServer, "Listen on given port")
```

Comment:
Minors - as above? Just `--port` and list defaults.

Reply:
Fixed parameter name, default already displayed by Cobra